README.md

```diff
@@ -23,6 +23,19 @@ This dataset was prepared by Pierre-Luc Carrier and Aaron Courville, as part of
 
 Past Kaggle competition for this partical challenge: https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge
 
+##To install
+--On Linux
```

Comment:
also make sure python3-dev is there, i had python-dev but apparently they're different